frontend/src/v5/ui/themes/overrides-mui.d.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright (C) 2021 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { CSSProperties } from 'react';
+import '@material-ui/core/styles';
+
+interface IGradient {
+	main: string;
+}
+
+interface IShadow {
```

Comment:
Referring to George's comment, we should remove the old shadows definitions, or replace it with a new ones if it is already final.

Reply:
Heads up, Michal, I rebased the branch, so if you are going to change something you should delete the local branch and fetch the new one to avoid conflicts without pain.

Reply:
@m1chalkubiak  @george-hadfield if the names of the shadows are ok but the values are the things that are being changed we can keep the old shadows for now.

Reply:
@sanmont3drepo In fact quite a lot has changed compared to what was originally implemented by me:
https://www.figma.com/file/djgxvRKOaMMyutswn5yywU/Teamspace?node-id=811%3A264
From what I can see, the only shadow we are using now is the one on the button focus state. Used to be `SHADOW.MEDIUM` now `SHADOW.6DP` or something. Altho I'm not sure as I have not received confirmation that these shadows versions are final. Any news regarding that @george-hadfield ?

Reply:
> @sanmont3drepo In fact quite a lot has changed compared to what was originally implemented by me:
> https://www.figma.com/file/djgxvRKOaMMyutswn5yywU/Teamspace?node-id=811%3A264
> From what I can see, the only shadow we are using now is the one on the button focus state. Used to be `SHADOW.MEDIUM` now `SHADOW.6DP` or something. Altho I'm not sure as I have not received confirmation that these shadows versions are final. Any news regarding that @george-hadfield ?

Hi @m1chalkubiak - I'm happy to stay with the existing shadow you have already implemented for now. Once it's built, then we can review then and see if it is worth changing.

Reply:
@george-hadfield It will never be very problematic to change these shadows, so we can stick to the old implementation for now. However, the main question as to whether we should add new shadows is whether the new shadows are used somewhere later in the application, I would assume they are.